indico/queries/datasets.py

```diff
@@ -262,7 +262,27 @@ def requests(self):
                 debouncer.backoff()
         yield GetDataset(id=dataset_id)
 
+       
+class RemoveFile(GraphQLRequest):
+    query = """
```

Comment:
Could you add pydoc comments to this, and run sphinx build if you're able? Otherwise just update the comments and I can run the build script. Otherwise, looks good! Thank you.